scss/tech.scss

```diff
@@ -0,0 +1,40 @@
+ @import 'color';
+
+.technoContent {
+  margin: 12px;
+  background: $color-green;
+  height: 300px;
+  width: 175px;
+  padding: 8.5px;
```

Comment:
Is this to center the content? Use flex `justify-content: center` to align horizontally and `align-items: center` to center vertically if thats what you want